app/controllers/application_controller.rb

```diff
@@ -2,4 +2,30 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(
+      :name,
+      :email,
+      :password,
+      :password_confirmation,
+      :remember_me
+    ) }
+    devise_parameter_sanitizer.for(:sign_in) { |u| u.permit(
+      :login,
+      :name,
+      :email,
+      :password,
+      :remember_me
+    ) }
+    devise_parameter_sanitizer.for(:account_update) { |u| u.permit(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.